site/content/en/docs/tasks/run_mpi_jobs.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Run A MPIJob"
+date: 2023-05-16
+weight: 6
+description: >
+  Run a Kueue scheduled MPIJob
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [MPI Operator](https://github.com/kubeflow/mpi-operator) MPIJobs.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+Check [administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial cluster setup.
+
+Check [the MPI Operator installation guide](https://github.com/kubeflow/mpi-operator#installation).
+
+## MPI Operator definition
+
+
+### a. Queue selection
+
+The target [local queue](/docs/concepts/local_queue) should be specified in the `metadata.labels` section of the MPIJob configuration.
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Set Suspend field in MPIOperator
+
+```yaml
+spec:
+  runPolicy:
+    suspend: true
+```
+
+### c. Set MPIJobs as an allowed workload in Kueue Configuration
```

Comment:
This should be in the "Before you begin" section.

Because not every user needs to do this, only the person configuring Kueue.
Also, link to https://kueue.sigs.k8s.io/docs/installation/#install-a-custom-configured-released-version

Reply:
Updated.